docs/central-users.rst

```diff
@@ -226,26 +227,32 @@ To manage App Users, navigate to the project whose App Users you wish to manage,
 Creating an App User
 ~~~~~~~~~~~~~~~~~~~~
 
-To create a new App User, click on the :guilabel:`Create app user` button on the right side of the App Users listing page. You will see a popup that looks like this:
+To create a new App User, click on the :guilabel:`Create app user` button on the right side of the App Users listing page.
+
+First, you will need to provide a Display Name for the new user. This name will appear in data tables, exports, and system logs.
+
+Ideally, every individual involved in data collection gets their own App User account. This makes it easiest to track who did what and to ensure that only authorized individuals are submitting data. In that case, the App User Display Names are the names of the individuals. However, this is not always practical, especially if you have a lot of data collectors. If you have a lot of data collectors and want to give them each their own App User account, consider using a script such as `this pyodk-based one <https://getodk.github.io/pyodk/examples/app_user_provisioner/app_user_provisioner/>`_ to create App Users and manage form assignments.
 
-.. image:: /img/central-users/app-users-create.png
+Alternatively, App Users can represent devices (device1, device2) or roles (Hospital1 Nurse, Lab Tech). If using these options, make sure you know how you will track what individual users are doing and ensure that settings QR codes remain private.
```

Comment:
If using these options, make sure you know how you will track what individual users are doing. One easy way to do this is by adding metadata (/form-question-types/#metadata) like deviceID or enumerator name as a question (with a last-saved enabled (form-logic/#values-from-the-last-saved-record)).

You'll also need to ensure that settings QR codes remain private.